src/geo/projection/globe.js

```diff
@@ -164,6 +164,11 @@ export default class Globe extends Mercator {
         return new MercatorCoordinate(mx, my);
     }
 
+    pointCoordinate3D(tr: Transform, x: number, y: number): ?Vec4 {
+        const mc = this.pointCoordinate(tr, x, y, 0);
```

Comment:
Logging offline discussion:
Terrain.pointCoordinate raycasting is implemented only for Mercator projection and is acceptable implementation constraint.